crates/metrics/src/core_metrics.rs

```diff
@@ -0,0 +1,60 @@
+//! Core pool metrics.
+// This is an example of metrics implementation.
+
+use std::sync::atomic::{AtomicU64, Ordering};
+use metrics::{absolute_counter, Label};
+use crate::report_metrics::ReportMetrics;
+
+pub struct PoolMetrics {
```

Comment:
The tx pool isn't being implemented properly right now so I don't think we can extract any useful metrics out of it. Fwiw, tx pool metric shouldn't be in the main `metrics` crate anyway as it's a katana thing only. Let's remove this for now.